<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>License_imp_base_dialog</class>
 <widget class="QMainWindow" name="License_imp_base_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>授权文件导入</string>
  </property>
  <property name="windowIcon">
   <iconset resource="license_import.qrc">
    <normaloff>:/res/import.ico</normaloff>:/res/import.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>29</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>授权文件导入工具</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>授权文件路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pFile_path_lineedit"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_pFile_path_button">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>路径选择</string>
        </property>
        <property name="icon">
         <iconset resource="license_import.qrc">
          <normaloff>:/res/folder.png</normaloff>:/res/folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="m_pCheck_button">
        <property name="text">
         <string>查看</string>
        </property>
        <property name="icon">
         <iconset resource="license_import.qrc">
          <normaloff>:/res/list.png</normaloff>:/res/list.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pImp_button">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="license_import.qrc">
          <normaloff>:/res/ok.png</normaloff>:/res/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pClose_button">
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="icon">
         <iconset resource="license_import.qrc">
          <normaloff>:/res/cancel.png</normaloff>:/res/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="m_pLicense_info_table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>170</number>
      </attribute>
      <column>
       <property name="text">
        <string>描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>值</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="license_import.qrc"/>
 </resources>
 <connections/>
</ui>
